mlflow/cli.py

```diff
@@ -28,7 +29,7 @@
 
 
 @click.group()
-@click.version_option()
+@click.version_option(version=version.VERSION)
```

Comment:
This wasn't working before due to the process that click uses to infer version:

1. If version is provided, return string.
2. If version is not provided, utilize importlib.metadata.version() to get the version from the package name.
      [This only works on Python>=3.8. If Python<3.8, the backport importlib_metadata must be installed (which we don't and I don't think it's worth it to include that)]
3. If package_name is not provided, a stack frame inspection will be conducted. _If this happens, the package name must match_ (which it doesn't, since we're calling an entry point of `mlflow` and the package name is `mlflow-skinny`. 

I figured that explicitly setting the version string here is the better approach to get this to work.

Reply:
For reference: https://click.palletsprojects.com/en/8.1.x/api/#click.version_option